Bottom: df19e48c737e9b2ce1d31c7d5a595a3adc2154b8
Top:    db7fa6c2e75c92ee4edfce2e95bffd067450ac65
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-12 01:49:47 +0100

hid/common: Remove #include "draw.h" from hidgl.c (twice!)


---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 6bd48c3..0bbd621 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -44,11 +44,9 @@
 #include "action.h"
 #include "crosshair.h"
 #include "data.h"
-#include "draw.h"
 #include "error.h"
 #include "global.h"
 #include "mymem.h"
-#include "draw.h"
 #include "clip.h"
 
 #include "hid.h"
